app/serializers/note_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class NoteSerializer < ActiveModel::Serializer
+  attributes :id, :title
+  attribute :note_type, key: :type
+  attributes :word_count, :created_at, :content, :content_length
+  belongs_to :user, serializer: UserSerializer
```

Comment:
```suggestion
  belongs_to :user
```
De nuevo, creo que aplica el mismo caso.